### ISSUE TITLE
History annotations are missing in generated history schema

### ISSUE THREAD

**Issue by jan-4brainsolutions** [flume/enthistory#33]
**Describe the bug**
When generating the load.Schema structs for the history tables, the annotations are set as
```
historySchema.Annotations = map[string]any{
	"EntSQL": map[string]any{
		"table": info.TableName,
	},
}
```

**To Reproduce**
Generate history tables.

**Expected behavior**
I would expect the load.Schema model to exactly match the entity files for history tables, which include Annotations for this library.

**Additional context**
Under certain circumstances, This results in FooHistoryHistory tables being generated when using the schema/snapshot ent feature flag.

**Comment by frisbm** [comment]
@jan-4brainsolutions thanks for reporting the bug, I think i'm following the annotations part of this and can fix this to reconcile the annotations in the schemas but I just want to check about the `schema/snapshot` part of this issue. Is that related or is it a separate bug?

**Comment by jan-4brainsolutions** [comment]
It is related. I added more information in this pull request (which still needs further testing) https://github.com/flume/enthistory/pull/34

**Comment by frisbm** [comment]
Closing as resolved by #34
